src/staging/src/components/search/mainSearch.js

```diff
@@ -58,6 +58,9 @@ const MainSearch = () => {
           Search
         </Button>
       </div>
+      <div className="mt-1">
+        <i className={"fa fa-info-circle"}></i> <em>Park search is limited during beta</em>
```

Comment:
TIL `className={"fa fa-info-circle"}` and`className="fa fa-info-circle"` are equivalent 🙂

Reply:
Haha yeah that was a lazy copy&paste on my part.  The code I copied from had a variable concatenated on there.

I'll create an update

Reply:
I don't think you need to, it's just a style difference. I thought braces added extra processing but apparently not